examples/dlccli/index.ts

```diff
@@ -82,18 +83,19 @@ async function ProposeContractToBob(offerMessage: OfferMessage) {
   return values[name];
 }
 
-async function DecideOutcome(): Promise<{ message: string; index: number }> {
+async function DecideOutcome(): Promise<{ message_string: string; index: number }> {
   const name = "Outcome choice";
   const answers = await inquirer.prompt([
     {
       name,
       message: "Choose the outcome of the event",
       type: "list",
-      choices: contract.messages.map((message, index) => {
+      choices: contract.messagesList.map((message, index) => {
+        const message_string = message['messages'][0];
```

Comment:
You don't need to do `['messages']`, you can simply do `.messages` instead. Otherwise I think it's ok, if you wanted to enable inputing several string per outcomes you'd need to also change the input part, but I think it's a bit an overkill for an example.

Reply:
Cool catch i was testing index notation instead of dot notation when trying to fix something else and forgot to set it back.